Default (Windows).sublime-keymap

```diff
@@ -1,13 +1,13 @@
 [
   // Find the dependents for the current module
   {
-    "keys": ["control+alt+up"],
+    "keys": ["ctrl+shift+up"],
```

Comment:
ctrl is what is actually on a linux/windows keyboard and not the word
"control".
I wish we could be uniform!   Alas Steve Jobs had to "Think Different".
:-).
I also changed the order of osx commands more because of the actual key
locations (left to right) rather than some OSX semantics....likewise I must
defer to you on OSX.

On Wed, Mar 11, 2015 at 7:45 PM, Joel Kemp <notifications@github.com> wrote:

> In Default (Windows).sublime-keymap
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26274702>:
>
> > @@ -1,13 +1,13 @@
> >  [
> >    // Find the dependents for the current module
> >    {
> > -    "keys": ["control+alt+up"],
> > +    "keys": ["ctrl+shift+up"],
>
> I really wanted uniformity about the key bindings, but as an OSX user,
> I'll defer to your judgement on more ergonomic key bindings.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26274702>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David